core/rfb.js

```diff
@@ -36,6 +38,12 @@ import TightPNGDecoder from "./decoders/tightpng.js";
 const DISCONNECT_TIMEOUT = 3;
 const DEFAULT_BACKGROUND = 'rgb(40, 40, 40)';
 
+// Gesture thresholds
+const ZOOMSENS = 30;
+const SCRLSENS = 50;
```

Comment:
These two should get some sort of prefix, like `GESTURE_ZOOMSENS`

Reply:
And do we use all-caps on other constants?

Reply:
It seems like it

fixed in 6d57ee9